render-configs.sh

```diff
@@ -1,32 +1,43 @@
 #!/bin/bash
 
-LOCAL_TOKEN=$(cat ~/.vault-token)
-VAULT_TOKEN=${1:-$LOCAL_TOKEN}
+# This script pulls the needed secrets from vault and sets the values as environment variables
+# If you want these values to show up in intellij test/run profiles, you need to restart intellij after running this script
+
+# If you want to run Azure Integration tests locally, you need to point to the "integration" environment
+AZURE_ENV=${1:-dev}
 
 vault read -format=json secret/dsde/datarepo/dev/sa-key.json \
     | jq .data | tee /tmp/jade-dev-account.json \
     | jq -r .private_key > /tmp/jade-dev-account.pem
 
-vault read -field=tenant-id secret/dsde/datarepo/dev/azure-application-secrets \
-    > /tmp/jade-dev-tenant-id.key
+vault read -field=tenant-id secret/dsde/datarepo/${AZURE_ENV}/azure-application-secrets \
```

Comment:
Could you run `shellcheck` on this script? I think these env vars should have double quotes around them.
```suggestion
vault read -field=tenant-id secret/dsde/datarepo/"${AZURE_ENV}"/azure-application-secrets \
```